docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,93 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+packageName: '@mui/base'
+---
+
+# Select
+
+<p class="description">The SelectUnstyled and MultiSelectUnstyled components let you create menus of options for users to choose from.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+MUI Base offers two components to replace the native HTML `<select>` tag: `SelectUnstyled` and `MultiSelectUnstyled`.
+
+## SelectUnstyled
+
+```tsx
+import SelectUnstyled from '@mui/base/SelectUnstyled';
+```
+
+### Basic usage
+
+{{"demo": "UnstyledSelectSimple.js", "defaultCodeOpen": false}}
+
+The `SelectUnstyled` component accepts generic props.
+Due to TypeScript limitations, this may cause unexpected behavior when wrapping the component in `forwardRef` (or other higher-order components).
+In such cases, the generic argument will be defaulted to `unknown` and type suggestions will be incomplete.
+To avoid this, you can manually cast the resulting component to the correct type, as shown in the demo above.
+
+For the sake of brevity, the rest of the demos that follow will not use `forwardRef`.
+
+### Controlled select
+
+`SelectUnstyled` can be used as an uncontrolled (as shown in the demo above) or controlled component.
+
+{{"demo": "UnstyledSelectControlled.js", "defaultCodeOpen": false}}
+
+### Object values
+
+The `UnstyledSelect` component can be used with non-string values.
+
+{{"demo": "UnstyledSelectObjectValues.js", "defaultCodeOpen": false}}
+
+### Selected value appearance
+
+You can customize the selected value display by providing a function to the `renderValue` prop.
+The element returned by this function will be rendered inside the select's button.
+
+{{"demo": "UnstyledSelectCustomRenderValue.js", "defaultCodeOpen": false}}
+
+### Option appearance
+
+Options don't have to be plain strings.
+You can include custom elements to be rendered inside the listbox.
+
+{{"demo": "UnstyledSelectRichOptions.js", "defaultCodeOpen": false}}
+
+### Grouping options
+
+Options can be grouped, similarly to the how the native `select` element works.
+Unlike the native `select`, however, groups can be nested.
+
+Place the `Option` components inside `OptionGroup` to achieve this.
+
+{{"demo": "UnstyledSelectGrouping.js", "defaultCodeOpen": false}}
+
+## MultiSelectUnstyled
+
+The `MultiSelectUnstyled` component lets your users select multiple options.
+
+```js
+import { MultiSelectUnstyled } from '@mui/base/SelectUnstyled';
+```
+
+{{"demo": "UnstyledSelectMultiple.js", "defaultCodeOpen": false}}
+
+## The useSelect hook
```

Comment:
```suggestion
## The `useSelect` hook
```

?

Reply:
I think we want to avoid back ticks in headers. At least, I think that's the way it already is elsewhere in the docs. But maybe we don't have a consistent pattern established.